test/unit/test_mortars.py

```diff
@@ -888,16 +888,16 @@ def test_replace_1d_with_identity(self):
         gb = self.setup_bucket(pert=False)
         mg1, mg2 = self._mortar_grids(gb)
 
-        proj_1_h = mg1.master_to_mortar_int().copy()
-        proj_1_l = mg1.slave_to_mortar_int().copy()
+        proj_1_h = mg1.primary_to_mortar_int().copy()
```

Comment:
h and l are primary and secondary

Reply:
A potential candidate for another PR as well.

Reply:
TODO: See #478